Getting-Started/Design/Templates/index.md

```diff
@@ -85,8 +85,15 @@ By default, when defining a section it is required. To make the section optional
 
     @RenderSection("Head", required: false)
 
- On your child page call `@section Head {}` and then type your markup that will be pushed into the Master Template.  
-
+On your child page call `@section Head {}` and then type your markup that will be pushed into the Master Template:
```

Comment:
Should we tweak this text at the same time? for clarity.... so it's 

On your child page template call...

or on your inheriting page template call...

Reply:
That is an excellent observation @umohl and yes we should!, thinking reading the text, should we at the same time tweak 'On your child page' to be clear its an inheriting template where  you place the section... (or is it obvious?)

Let me know what you think...

then we can pull this in, if not I'll pull it in anyway because the example is great, and required here.

Reply:
I think "On your child page template call ..." is gives a fine explanation, really. Inheritance is already mentioned multiple times above the section in question, so I think it's obvious.

Reply:
cool, is it easy for you to update your PR to add the word 'template'? or I can drop it in afters...